app/src/main/res/values-da-rDK/strings.xml

```diff
@@ -1055,7 +1055,7 @@ Skift til denne konto?</string>
   <string name="then_done_highlight">dernæst Færdig</string>
   <string name="for_your_security_be_sure_to_delete_your_saved_password_file">Sørg af sikkerhedshensyn for at slette den gemte adgangskodefil.</string>
   <string name="delete_your_saved_password_file">slet den gemte adgangskodefil.</string>
-  <string name="need_help_checkout_out_import_help">Behov for hjælp? Tjek import-hjælp ud.</string>
+  <string name="need_help_check_out_import_help">Behov for hjælp? Tjek import-hjælp ud.</string>
```

Comment:
You do not need to update these other files, just the base one.

The next Crowdin sync will take care of the rest.

Reply:
So I rolled this back, it does seem to be what is failing the build. Cause the that version of the string no longer exists in the default version. Are we ok to merge, crowdin is a black box to me.

Reply:
Hmmm, @SaintPatrck do we have a good way of doing this?

Reply:
We can ignore `ExtraTranslation` since Crowdin will be managing this. I opened #4176 to ignore it.